src/afrey/boot_asset_fingerprint.clj

```diff
@@ -27,8 +36,9 @@
                           (core/output-files)
                           (core/by-ext (or extensions [".html"])))
             file-hashes (into {}
-                          (map (juxt :path :hash))
-                          (core/output-files fileset))]
+                              (map (juxt :path :hash))
+                              (core/by-ext (or rename-extensions [".js" ".css"]) (core/output-files fileset)))
```

Comment:
What's the reasoning for adding the rename-extensions list? Is it a performance tweak, because of the new `file-rename-hash` calculation? I worry that if someone adds the `${}` selector around, say an `img` resource, but they forget to add `jpeg` to the extension list, they will be confused when their resource isn't found.